qutebrowser/browser/webengine/webview.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, *, tabdata, win_id, private, parent=None):
 
         theme_color = self.style().standardPalette().color(QPalette.Base)
         if private:
+            assert webenginesettings.private_profile is not None
```

Comment:
FWIW the `profile.isOffTheRecord()` below would crash anyways if it was `None`, but I guess it doesn't hurt to make it a bit more explicit.